### ISSUE TITLE
Broken

### ISSUE THREAD

**Issue by jerycho007** [TimidScript/UserScripts#18]
Greetings,
I'd like to ask if the image viewer works for anybody since the new Firefox update because for me it is broken.

**Comment by TimidScript** [comment]
Yeah it seems broken. FireFox is becoming a pain and a bit of bloatware. It was better when they released one or two a year instead of these quick successive releases. 

I will try to fix it after the holidays next year.  Happy early Xmas and new year.

**Comment by jerycho007** [comment]
Update pretty please?

**Comment by TimidScript** [comment]
I am sorry, due to personal issues I have been unable to work on anything for quite sometime. I do not know when I can get back to scripting. Thank you for understanding.
